doc/source/changes/version_0_26.rst.inc

```diff
@@ -255,4 +255,7 @@ Fixes
 * fixed array creation with axis(es) given as string containing only one label (axis name and label were inversed).
 
 * fixed original file being deleted when trying to overwrite a file via `Session.save` or `open_excel` failed
-  (closes :issue:`441`)
\ No newline at end of file
+  (closes :issue:`441`)
+
+* fixed loading arrays from Excel sheets containing blank or invisible cells on right column or bottom line
```

Comment:
now that I think of it, "invisible cells" seems weird. The cells themselves are visible, their content might not. But I guess simply using "blank cells" should be clear enough, no?

Reply:
=> "containing blank cells below or right of the array to read"?